backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/material/cockpit/rowfactory/MainRowBucketId.java

```diff
@@ -70,20 +68,13 @@ public static MainRowBucketId createPlainInstance(@NonNull final ProductId produ
 
 	ProductId productId;
 	LocalDate date;
-
-	private BigDecimal pmmQtyPromised = BigDecimal.ZERO;
-
-	private BigDecimal qtyReserved = BigDecimal.ZERO;
-
-	private BigDecimal qtyOrdered = BigDecimal.ZERO;
-
-	private BigDecimal qtyMaterialentnahme = BigDecimal.ZERO;
-
-	private BigDecimal qtyMrp = BigDecimal.ZERO;
-
-	private BigDecimal qtyPromised = BigDecimal.ZERO;
-
-	private BigDecimal qtyOnHand = BigDecimal.ZERO;
+	BigDecimal pmmQtyPromised = BigDecimal.ZERO;
```

Comment:
imho, all these qtys can be removed because
* they are not used
* they are already removed in master